src/components/ManageTransactions/index.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useEffect } from 'react';
+import { useTransactionContext } from '../../context/globalState';
+import TransactionItem from '../TransactionItem';
+import TransactionForm from '../TransactionForm';
+import { filterTransactions } from './utils';
+import { StyledTransactionLabel, StyledTransactionInput } from '../TransactionForm/styled';
+import {
+  StyledManageTransactionsWrapper,
+  StyledManageTransactions,
+  StyledManageTransactionsForm,
+} from './styled';
+
+const ManageTransactions = () => {
```

Comment:
no comments on except that the component file is too long
would be great if you divide it into 2 smaller components (remember the file structure)
```
ManageTransactions
|-index.js
|-styled.js
|-utils.js
|-comps
   |-index.js (file importing and exporting exporting all the sub comps )
   |-SubComName (folder having the usual structure)
   |- AnotherSubComp
```

Reply:
I propose to do it another way.
The filtered transactions part is almost the same as the `TransactionHistory` component.
And I can create a new component which is called `TransactionFilters`.
So the `ManageTransaction` component would look like:
```
<StyledManageTransactionsWrapper>
  <TransactionFilters />
  <TransactionHistory />
</StyledManageTransactionsWrapper>
```
I can rename `TransactionHistory` to `TransactionList` for example if it sounds better.

Reply:
this wouldn't work.. since transaction history is getting its data from the context, regardless of filters 
right?

Reply:
a suggestion to solve this.. you can take out the react elements part from transactionhistory and make a dumb component from it, gets data and renders it .. no logic in it